contentful/content-migrations/README.md

```diff
@@ -13,11 +13,13 @@ Guidance for making modifications to content types and updating entries programa
 Currently migrations are run manually, there will likely be a follow on piece to run these as part of a pipeline
 
 1. Add your migration script following the convention of `YYYYMMDD-HHMM-description-of-migration.js`
-2. run the migration script
+2. **Ensure you've disabled the Contentful Webhook** so that changes won't be posted during the migration or stacked up and posted when it's turned back on
```

Comment:
Is it worth adding more specific instructions? E.g. `Disable function _function name_ on the relevant environment`?

Reply:
Good idea, thanks - this update look better?